content/dart/concepts/list/list.md

```diff
@@ -0,0 +1,155 @@
+```
+
+## Different List Types
+
+There are two types of lists in Dart, fixed-length and growable. A fixed-length list has a specified size that cannot be changed after initialization. When creating a fixed-length list, it starts empty and needs items to be added to it. It is useful when the number of items in a list is known in advance. The following code block demonstrates the initialization of a fixed-length list:
+
+```dart
+// Example of a fixed length list
+var fixedLengthList = List(3);
+```
+
+>**Note:** If more than the specified number of items are added, it will result in an error.
+
+When creating a fixed-length list, it starts empty and needs items added to it. Fixed-length lists are useful when a known limit on list items is required ahead of time. This enforces the list to stay at the specified limit.
+
+```dart
+// Example of a growable list
+var growableList = ['Lion', 'Tiger', 'Leopard'];
+```
+
+## List Methods
+
+Lists are very versatile sets of data and Dart provides ways to manage the data held within the lists called methods. These methods include things like adding new data and removing data.
+
+### Adding Data
+
+Adding items of data to lists utilizing the following methods:
+- `(listname).add(item);` - This adds an item to the end of the list
+- `(listname).addAll(items);` - This adds multiple items to the end of the list
+
+```dart
+// Adding items to a list
+var ages = [21, 23, 34, 19, 45];
+
+ages.add(42); // Adds 42 to the end of the list
+ages.addAll(42, 37, 62); // Adds 42, 37 and 62 to the end of the list
+```
+
+It is also possible to add items at specific points within a list. This can be done by calling the list and specifying where the item should be added.
+
+```dart
+// Adding an item to a specific position within a list
+var ages = [21, 23, 34, 19, 45];
+
+ages[0] = 87; // Adds 87 to the first position of the list
+```
+This comes in handy with fixed length lists:
+
+```dart
+var fixedLengthList = List(3);
+
+fixedLengthList[0] = 1; // Sets the 1st item in the list as 1
+fixedLengthList[1] = 2; // Sets the 2nd item in the list as 2
+fixedLengthList[2] = 3; // Sets the 3rd item in the list as 3
+```
+
+### Removing Data
+
+Removing items of data from lists utilizes the following methods:
+- `(listname).remove(item);` - This removes the first occurance of the specified item
+- `(listname).removeAt(1);` - This removes the item at the specified index
+- `(listname).removeLast();` - This removes the last item in the list
+
+```dart
+var favouriteColors = ['red', 'red', 'orange', 'yellow', 'green'];
+
+favouriteColors.remove(red); // Removes the first 'red' in the list
+favouriteColors.removeAt(1); // Removes the second item in the list
+favouriteColors.removeLast(); // Removes the last item from the list
+```
+
+## Displaying List Data
+To display data held within a list the `print` method is used. This can be used to display the whole list, or specific items within the list.
```

Comment:
```suggestion

To display data held within a list the `print` method is used. This can be used to display the whole list, or specific items within the list.
```